src/integrations/intercom/services/conversations-getter.js

```diff
@@ -91,12 +90,12 @@ function ConversationsGetter(Implementation, params, opts, collectionName) {
 
                 return conversation;
               })
-              .then(conversations => [count, conversations]);
+              .then((conversationsFormatted) => [count, conversationsFormatted]);
           }))
         .catch((error) => {
           try {
             logger.error('Cannot access to Intercom conversations:', error.body.errors[0].message);
-          } catch (error) {
+          } catch (tryError) {
             logger.error('Cannot access to Intercom conversations:', error);
```

Comment:
It seems it there is a small change here. Suggestion:

```suggestion
            logger.error('Cannot access to Intercom conversations:', tryError);
```

Reply:
You're right the behaviour changes here.

I actually want to have the initial error in case of "unexpected" error not the one in the `try`/`catch` (that won't be interesting I think) for debug purposes.